Select_Flick_Week2/FavoriteMovieService.swift

```diff
@@ -10,23 +10,59 @@ import Foundation
 import RealmSwift
 
 class FavoriteMovieService {
+
+    static func getFavoriteMovies(username: String) -> UserFavoriteMoviesDTO {
+        let realm = try! Realm()
+        let predicate = NSPredicate(format: "username = %@", username)
+        let favoriteMovies = realm.objects(UserFavoriteMoviesDTO.self).filter(predicate)
+        if favoriteMovies.count == 0 {
+            return UserFavoriteMoviesDTO()
+        } else {
+            return (favoriteMovies.first)!
```

Comment:
i'm confused what this method returns, actually. why does it sometimes return the DTO and sometimes a movie?